### PR TITLE
Add a doc note that local_tmp was added in 2.1

### PR DESCRIPTION
**Pull request by alikins**
##### ISSUE TYPE
- Docs Pull Request
##### SUMMARY

Add a docs notes about the version local_tmp was added in.
As per https://github.com/ansible/ansible/issues/15719

### CODE REVIEW

**Comment by bcoca** [comment]
its 2.2 and probably better to use version_added construct

**Comment by alikins** [comment]
It's been a while, but we discussed this patch and decided that the option was added in 2.1.

File: docsite/rst/intro_configuration.rst [hunk 1]
```diff
@@ -468,6 +470,10 @@ If you'd like to change that, you can do so by altering this setting::
 
 Ansible will then choose a random directory name inside this location.
 
+.. note::
```
Comment (bcoca) [inline docsite/rst/intro_configuration.rst]: the note seems to be overkill as we already have version_added

**Comment by alikins** [comment]
updated and rebased

**Comment by abadger** [comment]
Thanks @alikins!

**Pull request merged**